com.unity.render-pipelines.core/Editor/Lighting/LightUI.Skin.cs

```diff
@@ -0,0 +1,42 @@
+using UnityEngine;
+
+namespace UnityEditor.Rendering
+{
+    /// <summary>
+    /// Contains a set of methods to help render the inspectors of Lights across SRP's
+    /// </summary>
+    public partial class LightUI
+    {
+        /// <summary>
+        /// Styles
+        /// </summary>
+        public static class Styles
+        {
+            public static readonly GUIContent generalHeader = EditorGUIUtility.TrTextContent("General");
+            public static readonly GUIContent shapeHeader = EditorGUIUtility.TrTextContent("Shape");
+            public static readonly GUIContent renderingHeader = EditorGUIUtility.TrTextContent("Rendering");
+            public static readonly GUIContent emissionHeader = EditorGUIUtility.TrTextContent("Emission");
+            public static readonly GUIContent shadowHeader = EditorGUIUtility.TrTextContent("Shadows");
+
+            public static readonly GUIContent lightLayer = EditorGUIUtility.TrTextContent("Light Layer", "Specifies the current Light Layers that the Light affects. This Light illuminates corresponding Renderers with the same Light Layer flags.");
+
+            // in casse that you want to keep the indentation but have nothing to write
+            public static readonly GUIContent empty = EditorGUIUtility.TrTextContent(" ");
```

Comment:
Is this the place for this empty style. Isn't this a more global thing? Or maybe I am missing something.

Reply:
moved in another PR